src/main/java/study03/model/Condition.java

```diff
@@ -0,0 +1,22 @@
+package study03.model;
+
+import java.util.Random;
+
+/*
+ *
+ * Condition
+ *
+ * @version 1.0.0
+ *
+ * 2021-11-10
+ *
+ * Copyright tnals1422
+ */
+public class Condition {
+
+    private static final int LIMIT = 10;
+
+    public static int generate() {
+        return new Random().nextInt(LIMIT);
+    }
```

Comment:
```suggestion
    private static final Random RANDOM = new Random();

    public static int generate() {
        return RANDOM.nextInt(LIMIT);
    }
```

Random 객체도 하나만 생성하고 재사용 가능해보이네요!

Reply:
할까말까 했던 부분들을 잘 짚어주셔서 좋아요